### PR TITLE
UKPE-1337 added selenium test for history buttons

### PR DESCRIPTION
**Pull request by andronz**

### CODE REVIEW

**Comment by Pawlyha** [comment]
+1

File: HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs [hunk 1]
```diff
@@ -175,47 +176,82 @@ public void Then_only_endpoints_with_chosen_parameters_should_be_shown()
...
             bool areStatusesCorrect = filteredStatuses.All(
-                m => m.Text.ToLower() == "faulty" || 
+                m => m.Text.ToLower() == "faulty" ||
                 m.Text.ToLower() == "healthy");
             Assert.True(areStatusesCorrect);
             Assert.True(filteredTags.Count(m => m.Text == tag) == filteredStatuses.Count);
             Assert.True(selectedTags.Any(m => string.Equals(m.Text, tag)));
         }
 
+        public void Then_endpoints_with_selected_filters_should_be_shown(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
```
Comment (Suremaker) [inline HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs]: This renders to: `5. THEN endpoints with selected filters should be shown [expectedCountOfSelectedTags: "1"] [expectedCountOfSelectedStatuses: "1"]`
Could you please rename method/method parameter names so the step description would be more natural?

File: HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.cs [hunk 2]
```diff
@@ -86,5 +87,27 @@ public void Applying_url_filter_to_endpoints()
                 _ => Then_only_endpoints_with_chosen_parameters_should_be_shown()
                 );
         }
+
+        [Scenario]
+        public void Filters_should_apply_when_traveling_forward_and_backward_on_history()
+        {
+            _driver.Manage().Timeouts().ImplicitlyWait(TimeSpan.FromSeconds(1));
```
Comment (Suremaker) [inline HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.cs]: Please hide the implementation details from the scenario.
It should be moved to either constructor (if applicable to all scenarios) or to some given/setup steps

File: HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs [hunk 3]
```diff
@@ -186,7 +186,7 @@ public void Then_only_endpoints_with_chosen_parameters_should_be_shown()
             Assert.True(selectedTags.Any(m => string.Equals(m.Text, tag)));
         }
 
-        public void Then_endpoints_with_selected_filters_should_be_shown(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
+        public void Then_endpoints_should_be_displayed_with_expected_count_of_filters(int expectedCountOfSelectedTags, int expectedCountOfSelectedStatuses)
```
Comment (Suremaker) [inline HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.Steps.cs]: Not much have changed here: `6. THEN endpoints should be displayed with expected count of filters [expectedCountOfSelectedTags: "1"] [expectedCountOfSelectedStatuses: "1"] `

Would it not be better to ensure that the step displays something like: `6. THEN endpoints should be filtered with 1 tag and 1 status filters`

`public void Then_endpoints_should_be_filtered_with_TAGFILTERS_tag_and_STATUSFILTERS_status_filters(int tagFilters, int statusFilters) {}`

[see lightbdd wiki](https://github.com/Suremaker/LightBDD/wiki/04-Formatting-Parameterized-Steps)

**Pull request merged**